abi/cpuid_amd64.go

```diff
@@ -0,0 +1,8 @@
+//go:build amd64
+// +build amd64
+
+package abi
+
```

Comment:
add `func asmCpuid(op uint32) (eax, ebx, ecx, edx uint32)` declaration and define

```go
func init() {
  cpuid = asmCpuid
}
```